build.sbt

```diff
@@ -2,32 +2,47 @@ import org.typelevel.scalacoptions.ScalacOptions
 import scoverage.ScoverageKeys
 import uk.gov.hmrc.DefaultBuildSettings
 
-ThisBuild / scalaVersion := "2.13.16"
+ThisBuild / scalaVersion := "3.3.5"
 ThisBuild / majorVersion := 0
-ThisBuild / semanticdbEnabled := true
-ThisBuild / semanticdbVersion := scalafixSemanticdb.revision
+
+val commonExcludedTpolecat = Set(
```

Comment:
Could you address https://github.com/hmrc/pillar2-stubs/pull/100/files#r2481031914 please? If you make the val lazy everything should be OK here.